agent/tool-scripts/postprocess/mpstat-postprocess

```diff
@@ -59,11 +70,67 @@ sub calc_timestamp {
 	return $date_ms + $this_time;
 }
 
-# read the mpstat.txt and collect stats for all non dm-* devices
-open(MPSTAT_TXT, "$dir/mpstat-stdout.txt") || die "could not find $dir/mpstat-stdout.txt\n";
+sub cpu_is_online {
+	my $cpu = shift;
+	if (exists $online_cpus{$cpu}) {
+		return $online_cpus{$cpu};
+	} elsif (-e $dir . "/online-cpus.txt" and open(FH, $dir . "/online-cpus.txt")) {
+		# Try a saved file of online status for this mpstat (this
+		# relies on this file being generated at some point before
+		# postprocessing, likely at stop-tools phase),
```

Comment:
Do we have this change yet?

Reply:
No, we do not have it in stop-tools.  Would you prefer this feature be included in this PR?

Reply:
I don't think we need to add that collection logic to `stop-tools` today, but having references to `online-cpus.txt` without an location in the code to generate it seems odd.

Reply:
Okay, added PR #1212 to collect this data in stop-tools and use it, enhancing the unit tests.